### PR TITLE
changed version to storm 1.0.2

### PR DESCRIPTION
**Pull request by aitorarce**
changed version to storm 1.0.2 and increased to 1Gb the assigned worker heap memory

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -13,7 +13,7 @@
   $user                                        = 'root',
   $group                                       = 'root',
   $home                                        = '/usr/lib/storm',
-  $version                                     = '1.0.0',
+  $version                                     = '1.0.2',
   $lib                                         = '/usr/lib/storm/lib',
```
Comment (alex-mon) [inline manifests/init.pp:17]: You can configure this things from your deployment code, this are all parameters that you can override.

File: manifests/init.pp [hunk 2]
```diff
@@ -39,7 +39,7 @@
   $config_file                                 = '/etc/storm/storm.yaml',
   $dev_zookeeper_path                          = '/tmp/dev-storm-zookeeper',
     #_ WORKERS _#
-  $worker_heap_memory_mb                       = 768,
+  $worker_heap_memory_mb                       = 1024,
```
Comment (alex-mon) [inline manifests/init.pp:42]: Same, no need to change it here.

File: manifests/init.pp (hunk 1 above)
Comment (aitorarce) [reply manifests/init.pp:17]: ok

File: manifests/init.pp (hunk 2 above)
Comment (aitorarce) [reply manifests/init.pp:42]: ok

**Pull request closed without merging**